access/ms-teams/config.go

```diff
@@ -32,6 +34,14 @@ func LoadConfig(filepath string) (*Config, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	// Azure secret format does not seem to support starting with a "/"
+	if strings.HasPrefix(conf.MSAPI.AppSecret, "/") {
```

Comment:
Is this the same format we require for loading from files for other plugins? I wonder why not make it explicit using e.g. `file://` scheme. It's ok if we do the same for other plugins.

Reply:
I remember seeing something in Teleport that used a leading `/` to distinguish a path to the file holding the content, as opposed to the literal content itself, so it's at least consistent with that. Although I can't seem to find it now, so maybe it's an obsolete idea,

Reply:
I stole the code from the slack plugin, in Teleport we are doing something similar, maybe a bit robust: https://github.com/gravitational/teleport/blob/c1c7fb8bb7656ebeb0d30446680dde8a0e408f3a/lib/utils/config.go#L32-L48